.changes/0.0.0.md

```diff
@@ -0,0 +1,3 @@
+## Previous Releases
+For information on prior major and minor releases, see their changelogs:
+
```

Comment:
I'm wondering if this footer for the 1.0.latest branch should actually say something like "previous versions of this package lived in `dbt-core`"?  @jtcohen6 do you have an opinion here?

Reply:
tried a hand at a message